### DISCUSSION TITLE
Unable to Log in to GitHub in Android Studio Bumblebee

### DISCUSSION THREAD

**Discussion by akshtshrma** [oppia/oppia-android#5618] [Q&A (Installation)] [answered]
### Operating System

Windows 11

### Android Studio Version

Bumblebee

### RAM size

8 GB

### Free disk space

477 GB

### Which step are you stuck on?

I’m encountering an issue while trying to log in to GitHub in Bumblebee. Every time I attempt to sign in, I receive an error, and the authentication process fails. @adhiamboperes Can you please help me in this issue.

### Error Log

```bash
Invalid authentication data
```

### Approaches already used to resolve the issue

I have attempted to log in using both GitHub tokens and the standard login method (username and password). Unfortunately, neither method works, and I am unable to log in.

**Comment by akshtshrma** [comment]
@adhiamboperes please help

**Comment by adhiamboperes** [comment, marked as answer]
Hi @akshtshrma, I checked the Jetbrains blog and it seems that there are issues using Github in older versions of IDEs, please see: https://intellij-support.jetbrains.com/hc/en-us/community/posts/19507482696082-Invalid-authentication-data-error. However, you can still use Git from within the terminal in your Android Studio to perform commit, push and pull operations. I found [this page](https://statistics.berkeley.edu/computing/faqs/git-auth) with instructions. PTAL and let me know if you run into further issues.

**Comment by akshtshrma** [comment]
@adhiamboperes, I’m facing some issues while updating the Gradle version. Is it possible to make changes to the project using Docker or by using the latest version of Android Studio (such as Koala)?

**Reply by adhiamboperes** [reply]
@akshtshrma, we are not able to update the gradle version. As of this week, when you pull the latest version of develop, we are phasing out Gradle and only able to build the app using Bazel. Please follow the instructions need to follow the instructions at https://github.com/oppia/oppia-android/wiki/Oppia-Bazel-Setup-Instructions.  to be able to use Bazel, and a newer AS version like Jellyfish.
